src/webhook/handlers/__tests__/__snapshots__/tutorial.test.js.snap

```diff
@@ -476,8 +476,8 @@ Object {
   "replies": Array [
     Object {
       "altText": "When I provide hoax-busting replies to you, I would like to ask you for any feedback on the crowd-sourced reply.
-In order to achieve that, I need to ask for your permission to \\"send your message to our chatroom\\".
-The permission will be used to send only this one message of yours back to this parti⋯⋯
+In order to achieve that, I need to ask for your permission to \\"Send messages to chats\\".
+The permission will be used to send only this one message of yours back to this particular chatr⋯⋯
```

Comment:
Text will ellipsis before assigning to altText, so the snapshot looks a little weird.